base/ltfssaxes.dtx

```diff
@@ -192,503 +248,3158 @@
+\DeclareFontSeriesChangeRule {ebec}{c}  {ebc}  {}
+\DeclareFontSeriesChangeRule {ebec}{sc} {ebsc} {}
+\DeclareFontSeriesChangeRule {ebec}{sx} {ebsx} {}
+\DeclareFontSeriesChangeRule {ebec}{x}  {ebx}  {}
+\DeclareFontSeriesChangeRule {ebec}{ex} {ebex} {}
+\DeclareFontSeriesChangeRule {ebec}{ux} {ebux} {}
+%    \end{macrocode}
+%
+%    \begin{macrocode}
+\DeclareFontSeriesChangeRule {ebc}{ul}  {ulc}  {}
+\DeclareFontSeriesChangeRule {ebc}{el}  {elc}  {}
+\DeclareFontSeriesChangeRule {ebc}{l}   {lc}   {}
+\DeclareFontSeriesChangeRule {ebc}{sl}  {slc}  {}
+\DeclareFontSeriesChangeRule {ebc}{sb}  {sbc}  {bc}
+\DeclareFontSeriesChangeRule {ebc}{b}   {bc}   {}
+\DeclareFontSeriesChangeRule {ebc}{eb}  {ebc}  {bc}
+\DeclareFontSeriesChangeRule {ebc}{ub}  {ubc}  {bc}
+\DeclareFontSeriesChangeRule {ebc}{uc}  {ebuc} {}
+\DeclareFontSeriesChangeRule {ebc}{ec}  {ebec} {}
+\DeclareFontSeriesChangeRule {ebc}{c}   {ebc}  {}
+\DeclareFontSeriesChangeRule {ebc}{sc}  {ebsc} {}
+\DeclareFontSeriesChangeRule {ebc}{sx}  {ebsx} {}
+\DeclareFontSeriesChangeRule {ebc}{x}   {ebx}  {}
+\DeclareFontSeriesChangeRule {ebc}{ex}  {ebex} {}
+\DeclareFontSeriesChangeRule {ebc}{ux}  {ebux} {}
+%    \end{macrocode}
+%
+%    \begin{macrocode}
+\DeclareFontSeriesChangeRule {ebsc}{ul} {ulsc} {}
+\DeclareFontSeriesChangeRule {ebsc}{el} {elsc} {}
+\DeclareFontSeriesChangeRule {ebsc}{l}  {lsc}  {}
+\DeclareFontSeriesChangeRule {ebsc}{sl} {slsc} {}
+\DeclareFontSeriesChangeRule {ebsc}{sb} {sbsc} {bsc}
+\DeclareFontSeriesChangeRule {ebsc}{b}  {bsc}  {}
+\DeclareFontSeriesChangeRule {ebsc}{eb} {ebsc} {bsc}
+\DeclareFontSeriesChangeRule {ebsc}{ub} {ubsc} {bsc}
+\DeclareFontSeriesChangeRule {ebsc}{uc} {ebuc} {}
+\DeclareFontSeriesChangeRule {ebsc}{ec} {ebec} {}
+\DeclareFontSeriesChangeRule {ebsc}{c}  {ebc}  {}
+\DeclareFontSeriesChangeRule {ebsc}{sc} {ebsc} {}
+\DeclareFontSeriesChangeRule {ebsc}{sx} {ebsx} {}
+\DeclareFontSeriesChangeRule {ebsc}{x}  {ebx}  {}
+\DeclareFontSeriesChangeRule {ebsc}{ex} {ebex} {}
+\DeclareFontSeriesChangeRule {ebsc}{ux} {ebux} {}
+%    \end{macrocode}
+%
+%    \begin{macrocode}
+\DeclareFontSeriesChangeRule {eb}{uc}   {ebuc} {}
+\DeclareFontSeriesChangeRule {eb}{ec}   {ebec} {}
+\DeclareFontSeriesChangeRule {eb}{c}    {ebc}  {}
+\DeclareFontSeriesChangeRule {eb}{sc}   {ebsc} {}
+\DeclareFontSeriesChangeRule {eb}{sx}   {ebsx} {}
+\DeclareFontSeriesChangeRule {eb}{x}    {ebx}  {}
+\DeclareFontSeriesChangeRule {eb}{ex}   {ebex} {}
+\DeclareFontSeriesChangeRule {eb}{ux}   {ebux} {}
+%    \end{macrocode}
+%
+%    \begin{macrocode}
+\DeclareFontSeriesChangeRule {ebsx}{ul} {ulsx} {}
+\DeclareFontSeriesChangeRule {ebsx}{el} {elsx} {}
+\DeclareFontSeriesChangeRule {ebsx}{l}  {lsx}  {}
+\DeclareFontSeriesChangeRule {ebsx}{sl} {slsx} {}
+\DeclareFontSeriesChangeRule {ebsx}{sb} {sbsx} {bsx}
+\DeclareFontSeriesChangeRule {ebsx}{b}  {bsx}  {}
+\DeclareFontSeriesChangeRule {ebsx}{eb} {ebsx} {bsx}
+\DeclareFontSeriesChangeRule {ebsx}{ub} {ubsx} {bsx}
```

Comment:
```suggestion
\DeclareFontSeriesChangeRule {ebsx}{ub} {ubsx} {ebsx}
```

I think it is better to use `eb` instead of `b` in the fourth argument: `eb` is a better approximation for `ub` and since the fourth argument is then identical to the first, we can assume that this font face actually exists.